t/t5601-clone.sh

```diff
@@ -68,6 +68,14 @@ test_expect_success 'clone respects GIT_WORK_TREE' '
 
 '
 
+test_expect_success MINGW,CASE_INSENSITIVE_FS 'core.worktree is not added due to path case' '
+
+	mkdir CAPITALCASE &&
+	lower_path=$( cygpath -w "$(pwd)/CAPITALCASE" | tr A-Z a-z ) &&
```

Comment:
Please remove the spaces after the opening, and before the closing brace.

Reply:
Oh, and why not simply using `$(pwd)/capitalcase`? It is sufficient for the case to differ only in part of the path, right?

Reply:
> Please remove the spaces after the opening, and before the closing brace.

I wanted to do that myself, but the rest of this file seems to use this style. If you still want it removed, I will do.

Reply:
> Please remove the spaces after the opening, and before the closing brace.

Copied that from line 170. This files seems to use both styles. Fixed as requested.

Reply:
> Oh, and why not simply using `$(pwd)/capitalcase`? It is sufficient for the case to differ only in part of the path, right?

Sorry, I don't quite understand. If you mean that `C:/` will be different anyway, I was worried that tests could be run from `\\SERVER\share`

Reply:
No, I meant that the `uppercase` will be different from `UPPERCASE` anyway.

Reply:
Indeed, I was too fixed on the idea that `c:/` causes the problem and wanted to test both, but that's probably an overkill. I have changed as suggested now.

Reply:
I also dropped `cygpath -w` part, it seems that cygwin converts paths automatically when starting git.exe